static/app/views/settings/components/defaultSearchBar.tsx

```diff
@@ -10,9 +10,6 @@ export type RenderSearch = React.ComponentProps<
 
 export const SearchWrapper = styled('div')`
   display: flex;
-  * {
-    width: 100%;
-  }
```

Comment:
Can this break other places?

Reply:
afaik this is only being used in two places, so I believe not